cxx/isce3/matchtemplate/pycuampcor/cuOffset.cpp

```diff
@@ -21,7 +21,7 @@ void  cudaKernel_maxloc2D(const float* const images, int2* maxloc, float* maxval
     const int imageSize = imageNX * imageNY;
 
     for (int bid = 0; bid < nImages; bid++) {
-        float my_maxval = std::numeric_limits<float>::min();
+        float my_maxval = -std::numeric_limits<float>::max();
```

Comment:
```suggestion
        float my_maxval = std::numeric_limits<float>::lowest();
```

I don't think the negative of the max value is guaranteed to be a valid float value, so let's instead explicitly request the lowest possible float.